.buildkite/pipeline.verify.yml

```diff
@@ -176,8 +176,11 @@ steps:
             - PULUMI_ACCESS_TOKEN
       - improbable-eng/metahook#v0.4.1:
           pre-command: |
-            # This is slightly hacky, but means we don't need to build packages locally first
-            git fetch origin rc
+            # This is slightly hacky, but means we don't need to build
+            # packages locally first. The `set-branches` and
+            # `--depth=1` are to account for us using shallow clones
+            git remote set-branches --add origin rc
+            git fetch --depth=1 origin rc
```

Comment:
Your Git is still messed up and pulling in other peoples' commits, and unfortunately I won't be accepting PRs until you get it figured out (see https://grapl-internal.slack.com/archives/C02JCBKS97V/p1638406536030500 )